### PR TITLE
Improve find

### PR DESCRIPTION
**Pull request by sergiors**
\cc @haskellcamargo

### CODE REVIEW

**Comment by sergiors** [comment]
lgtm

File: src/find.php [hunk 1]
```diff
@@ -14,16 +14,9 @@
...
-        };
-
-        $x = head($xss);
-        $xs = tail($xss);
-
-        return $pred($x)
-            ? $x
-            : find($pred, $xs);
+    return partial(function (callable $pred, array $xss) {
+        return ifelse(equals([]), always(null), function (array $xss) use ($pred) {
```
Comment (haskellcamargo) [inline src/find.php:18]: Wouldn't be better to make them standard? `ifElse` instead of `ifelse`.

**Review by haskellcamargo — CHANGES_REQUESTED** [review]

File: src/find.php (hunk 1 above)
Comment (sergiors) [reply src/find.php:18]: yep, I also need to rename other fns like `getin`-> `getIn`

Comment (sergiors) [reply src/find.php:18]: done!

**Review by haskellcamargo — APPROVED** [review]
LGTM

**Comment by sergiors** [comment]
self lgtm

**Pull request merged**